torch_tvm/compiler.cpp

```diff
@@ -52,7 +63,17 @@ std::vector<DLManagedTensorPtr> set_input(
 
 DLManagedTensorPtr createParamTensor(const IValue& param_val) {
   auto tensor = param_val.toTensor();
-  auto dl_tensor = torch_tvm::utils::allocAndCopyData(tensor);
+  DLManagedTensor* dl_tensor = nullptr;
+#ifdef TVM_TRAINING
```

Comment:
I wonder if this would be better suited as a runtime parameter? recompiling just to train seems a bit odd. Ideally we'd merge the code here anyway, right? (fixing up allocAndCopyData somehow)

Reply:
we have some issues with static initialization unfortunately that make it hard to use a runtime parameter